scripts/ubuntu_wayland_setup

```diff
@@ -17,7 +17,7 @@ LIBDRM=2.4.114
 # Packaged dependencies
 sudo apt update
 sudo apt install --no-install-recommends \
-    ninja-build libegl1-mesa-dev libgles2-mesa-dev libgbm-dev libinput-dev libxkbcommon-dev \
+    ninja-build libegl1-mesa-dev libgles2-mesa-dev libgbm-dev libinput-dev \
```

Comment:
libxkbcommon-dev isn't needed? huh

Reply:
This is because it's installed in the main ci build. If it's only needed for Wayland then it should be changed back to how it was.

Reply:
I think I remember the issue: `mypy` need it to install `xkbcommon`. If I split out the backends, when `mypy` runs on an x11 backend, installing that package will fail. This wouldn't have been an issue before because the wayland backend was always installed.

Reply:
Mupy doesnt install packages does it? Do you mean tox?

Reply:
Yes, the `mypy` environment run by `tox`. Doesn't that need the wayland stuff to be built?

Reply:
Yes i think so